ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/methods/CreateSideChain.java

```diff
@@ -34,6 +45,26 @@ public String getName() {
 
     @Override
     public JsonRpcResponse response(final JsonRpcRequest req) {
-        return new JsonRpcSuccessResponse(req.getId(), Quantity.create(1));
+        Web3j web3j = Web3j.build(new HttpService("https://rinkeby.infura.io/v3/" + DEFAULT_TOKEN));
```

Comment:
Infura is a collection of Ethereum nodes.
But we are right now inside of our own Ethereum node: Pantheon.
The whole point of what we want to do is use the currently running Pantheon to access the ERA.